src/pages/iou/IOUModal.js

```diff
@@ -0,0 +1,150 @@
+import IOUConfirmPage from './steps/IOUConfirmPage';
+import Header from '../../components/Header';
+import styles from '../../styles/styles';
+import Icon from '../../components/Icon';
+import {getPreferredCurrency} from '../../libs/actions/IOU';
+import {Close, BackArrow} from '../../components/Icon/Expensicons';
+
+/**
+ * IOU modal for requesting money and splitting bills.
+ */
+const propTypes = {
+    /* Onyx Props */
+
+    // Url currently in view
+    currentURL: PropTypes.string,
+};
+
+// Determines type of step to display within Modal, value provides the title for that page.
+const Steps = {
+    IOUAmount: 'Amount',
+    IOUParticipants: 'Participants',
+    IOUConfirm: 'Confirm',
+};
+
+const defaultProps = {
+    currentURL: '',
+};
+
+class IOUModal extends Component {
+    constructor(props) {
+        super(props);
+
+        this.navigateToPreviousStep = this.navigateToPreviousStep.bind(this);
+        this.navigateToNextStep = this.navigateToNextStep.bind(this);
+
+        this.state = {
+            steps: [Steps.IOUAmount, Steps.IOUParticipants, Steps.IOUConfirm],
+            currentStepIndex: 0,
+            currentStepTitle: Steps.IOUAmount,
+            hasMultipleParticipants: this.props.currentURL === ROUTES.IOU_BILL_SPLIT,
+        };
+    }
+
+    componentDidMount() {
+        getPreferredCurrency();
+    }
+
+    /**
+     * Navigate to the next IOU step if possible
+     */
+    navigateToPreviousStep() {
+        if (this.state.currentStepIndex <= 0) {
+            return;
+        }
+        this.setState(prevState => ({
+            currentStepIndex: prevState.currentStepIndex - 1,
+            currentStepTitle: prevState.steps[prevState.currentStepIndex - 1] || '',
+        }));
+    }
+
+    /**
+     * Navigate to the previous IOU step if possible
+     */
+    navigateToNextStep() {
+        if (this.state.currentStepIndex >= this.state.steps.length - 1) {
+            return;
+        }
+        this.setState(prevState => ({
+            currentStepIndex: prevState.currentStepIndex + 1,
+            currentStepTitle: prevState.steps[prevState.currentStepIndex + 1] || '',
+        }));
+    }
+
+    render() {
+        return (
+            <>
+                <View style={[styles.headerBar, true && styles.borderBottom]}>
+                    <View style={[
+                        styles.dFlex,
+                        styles.flexRow,
+                        styles.alignItemsCenter,
+                        styles.flexGrow1,
+                        styles.justifyContentBetween,
+                        styles.overflowHidden,
+                    ]}
+                    >
+                        {this.state.currentStepIndex > 0
+                        && (
+                            <TouchableOpacity
+                                onPress={this.navigateToPreviousStep}
+                                style={[styles.touchableButtonImage]}
```

Comment:
Right now the back button and X seem a little small, which could be a problem for bigger screens. Could we increase the size of the touch area? 

![Screen Shot 2021-03-08 at 12 27 57 PM](https://user-images.githubusercontent.com/24466196/110383502-39f3a400-8011-11eb-9091-4fd3801dba40.png)

Reply:
Is the current touch area what you have shown in the blue outlines? That should be about 40x40, which I think isn't terrible.